packages/manager/modules/billing/src/components/services-actions/index.js

```diff
@@ -0,0 +1,20 @@
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+import '@ovh-ux/ng-translate-async-loader';
```

Comment:
Don't you require this dep in your peer ones ?

Reply:
I forgot to add it, nice catch !